components/FaencyProvider.tsx

```diff
@@ -43,7 +43,7 @@ export const FaencyProvider: React.FC<{ primaryColor: PrimaryColor }> = ({
   return (
     <IdProvider>
       <>
-        {globalStyles}
+        {globalStyles()}
```

Comment:
Should we also put it before the return?

Reply:
@paulocfjunior I don't understand your question, could you explain?

Reply:
I think @paulo is saying calling `globalStyle` function before the `return` instead of putting it in the jsx (as in the doc https://stitches.dev/docs/api#globalcss).

IMO both are ok. with maybe a preference for putting it before the `return` statement since jsx position will not change anything.

Reply:
Yeah, before sounds good 👍

Reply:
My bad, I scanned the docs too quickly :joy: 
Thanks for your time folks

Reply:
Haha thx @paulo joining us as Faency contributors 😂  and sorry for the mention 🙏